vtex-apps/console
# Feature/time controller relative mode

#### What is the purpose of this pull request?
Add relative mode to time controllers and synchronize the controllers state with the url query string

#### What problem is this solving?
- The relative mode allows the user the define a time period relative to now. 
- The synchronization of controllers with the url query string improve the user navigation as it allows 
to define the controllers values from the query string.

#### How should this be manually tested?
At https://germano--vtex.myvtex.com/admin/console/metrics?appName=render-server&mode=relative do the following:

- Relative mode
1) Choose relative mode
2) Choose a range  
3) Click apply

- Query string
   There are some interactions to check:
  - The change of controllers change the query string
  - Change or include one valid param in the query string
  - Try a refresh, back, forward or a copy/paste of link

#### Screenshots or example usage
![time mode](https://user-images.githubusercontent.com/12701669/52735512-f7d11300-2fae-11e9-9afe-393c113ec625.png)

#### Types of changes
